Sources/ArcGISToolkit/Utility/CredentialInputView.swift

```diff
@@ -47,27 +47,23 @@ extension View {
 
 struct CredentialInputSheetView_Previews: PreviewProvider {
     static var previews: some View {
-        Text(
-            "test",
-            bundle: .toolkitModule,
-            comment: "A label for development purposes only. This is not user visible."
-        )
```

Comment:
Text is overkill for this preview - it is still completely functional without it.